src/pages/accounts/components/AccountDialog.tsx

```diff
@@ -0,0 +1,118 @@
+import React from "react";
+import { styled } from "@mui/material/styles";
+import {
+  Icon,
+  Grid,
+  Chip,
+  Stack,
+  Button,
+  Divider,
+  Typography,
+} from "@mui/material";
+import { useAppSelector } from "../../../app/common/hooks";
+import { AccountEditForm, AccountInfo, AdminActions } from ".";
+import Dialog from "../../../shared/GenericDialog";
+
+const TitleButton = styled(Button)(({ theme }) => ({
+  margin: theme.spacing(0.5),
+  borderRadius: theme.spacing(3),
+}));
+
+interface AccountDialogProps {
+  open: boolean;
+  handleClose: () => void;
+}
+
+export function AccountDialog({ open, handleClose }: AccountDialogProps) {
+  const [editMode, setEditMode] = React.useState<boolean>(false);
+
+  const { selectedAccount } = useAppSelector((state) => state.accountManager);
+
+  const handleDialogClose = () => {
+    setEditMode(false);
+    handleClose();
+  };
+
+  const FooterButtons = [
+    {
+      label: "Close",
+      onClick: handleDialogClose,
+      color: "inherit",
+      variant: "outlined",
+    },
+  ];
+
+  const EditFooterButtons = [
+    {
+      label: "Save",
+      onClick: () => {},
```

Comment:
why do we have an empty onclick?

Reply:
That is for editing an account, as you know, this is not integrated with the backend, so there is some features missing.